source/Ship.cpp

```diff
@@ -564,9 +549,12 @@ void Ship::FinishLoading(bool isNewInstance)
 	// Check that all the "equipped" weapons actually match what your ship
 	// has, and that they are truly weapons. Remove any excess weapons and
 	// warn if any non-weapon outfits are "installed" in a hardpoint.
+	auto equipped = armament.GetEquipped();
 	for(auto &it : equipped)
```

Comment:
```suggestion
	for(auto &it : armament.GetEquipped())
```

Reply:
equipped is also used on line 616

Reply:
Oh shoot, I didn't see cause I didn't expand the code. Whoops

Reply:
No problem. :)